resources/views/resources/index.blade.php

```diff
@@ -42,7 +42,7 @@ class="w-full max-w-md text-sm placeholder-gray-500 border-gray-300 rounded-lg s
 
     </x-slot>
 
-    @empty(! $metrics = $resource->metrics())
+    @empty(!($metrics = $resource->metrics()))
```

Comment:
What is this even doing? Running an `empty()` check on `! $metrics = $resource->metrics()`? Doesn't it make more sense to just do `@if($metrics = $resource->metrics())`?

Reply:
Was already there, didn't touch this. But will fix